infrastructure/README.md

```diff
@@ -1,3 +1,5 @@
+## Infrastructure
+
 ### Installing dependencies
 ```bash
 $ brew install minikube kubectl
```

Comment:
`brew install minikube` failed for me and then I saw this: https://github.com/kubernetes/minikube/issues/1449

I guess it should be `brew cask install minikube && brew install kubectl` ?

Reply:
Fixed

Reply:
Fixed